openapi/paths/tags@{tag}@aml-checks@{amlCheckId}.yaml

```
@@ -0,0 +1,44 @@
+parameters:
+  - $ref: ../components/parameters/tag.yaml
+  - name: amlCheckId
+    in: path
+    description: ID of the AML check to tag.
+    required: true
+    schema:
+      type: string
+post:
+  x-products:
+    - Core
+  tags:
+    - Tags
+  summary: Tag an AML check
+  operationId: PostTagAmlCheck
+  x-sdk-operation-name: tagAmlCheck
+  description: Tags an AML check.
```

suggestion: should we add the information from the other descriptions here also?
```suggestion
  description: |-
    Tags an AML check.
    
    If a AML check in the list does not have the specified tag applied, the AML check is ignored.
    This is an asynchronous operation.
```